Packs/FeedDHS/Integrations/DHSFeedV2/DHSFeedV2.py

```diff
@@ -0,0 +1,223 @@
+        outputs_key_field='value',
+        outputs=indicators,
+        raw_response=indicators
+    )
+
+
+def get_collections_command(client: Taxii2FeedClient):
+    """
+    Get the available collections in the DHS server
+    """
+    collections = list()
+    for collection in client.collections:
+        collections.append({'Name': collection.title, 'ID': collection.id})
+    return CommandResults(
+        outputs_prefix='DHS.Collections',
+        outputs_key_field='ID',
+        outputs=collections,
+        readable_output=tableToMarkdown('DHS Server Collections:', collections))
+
+
+''' MAIN FUNCTION '''
+
+
+def main():
+    params = demisto.params()
+    args = demisto.args()
+
+    url = params.get('url', 'https://ais2.cisa.dhs.gov/taxii2/')
+    key = params.get('key')
+    certificate = params.get('certificate')
+    verify_certificate = not params.get('insecure', False)
+    proxies = handle_proxy()
+
+    collection_to_fetch = params.get('collection_to_fetch')
+    skip_complex_mode = COMPLEX_OBSERVATION_MODE_SKIP == params.get('observation_operator_mode')
+    feed_tags = argToList(params.get('feedTags'))
+    tlp_color = params.get('tlp_color', '')
+
+    initial_interval = params.get('initial_interval', '24 hours')
+    fetch_full_feed = params.get('fetch_full_feed') or False
+    is_incremental_feed = params.get('feedIncremental') or False
+    limit = arg_to_number(params.get('limit') or -1)
+    limit_per_request = arg_to_number(params.get('limit_per_request'))
+    objects_types = ['report', 'indicator', 'malware', 'campaign', 'attack-pattern',
+                     'course-of-action', 'intrusion-set', 'tool', 'threat-actor', 'infrastructure']
+    objects_to_fetch = argToList(params.get('objects_to_fetch') or objects_types)
+    default_api_root = params.get('default_api_root', 'public')
+
+    command = demisto.command()
+    demisto.info(f'Command being called is {command}')
+    commands = {
+        'dhs-get-indicators': get_indicators_command,
+        'dhs-get-collections': get_collections_command,
+    }
+
+    try:
+        assert_incremental_feed_params(fetch_full_feed, is_incremental_feed)
+        client = Taxii2FeedClient(
+            url=url,
+            collection_to_fetch=collection_to_fetch,
+            proxies=proxies,
+            verify=verify_certificate,
+            objects_to_fetch=objects_to_fetch,
+            skip_complex_mode=skip_complex_mode,
+            tags=feed_tags,
+            limit_per_request=limit_per_request,
+            tlp_color=tlp_color,
+            certificate=certificate,
+            key=key,
+            default_api_root=default_api_root,
+        )
+        client.initialise()
+
+        if command == 'test-module':
+            return_results(command_test_module(client, limit, fetch_full_feed))
+
+        elif command == 'fetch-indicators':
+            if fetch_full_feed:
+                limit = -1
+
+            last_run_indicators = get_feed_last_run()
```

Comment:
To be honest, the `get_feed_last_run` function is not needed anymore, and can be deprecated - please use `demisto.getLastRun()`.